expand_template

sevki opened the pull request.

sevki committed:
files: return a source file if not in a workspace

Signed-off-by: Sevki <s@sevki.org>

sevki committed:
label: make label of type skylark.String

Signed-off-by: Sevki <s@sevki.org>

sevki committed:
Merge branch 'master' into template

bldybuild commented:
Commits f987c003f012d80259af1e4e1513d0040f9ab851 and bc8b0713b5ffdf25e1d2ca7a582d39b079c05dca are not signed-off.

Please fix these issues:

>	1. Sevki has not signed-off f987c00.
>	2. Sevki has not signed-off bc8b071.


If you'd like more information on how to sign your commits please visit [signed-off-by.me](https://signed-off-by.me)

sevki committed:
skylark: unpack struct

this change makes bldy return a nicer message instead of panicing if value can't
be set

Signed-off-by: Sevki <s@sevki.org>

sevki committed:
Merge branch 'template' of github.com:sevki/build into template

sevki committed:
skylark: expand template support

https://docs.bazel.build/versions/master/skylark/lib/actions.html#expand_template

Signed-off-by: Sevki <s@sevki.org>
